internal/domain/macros.go

```diff
@@ -67,7 +69,9 @@ func NewMacro(release Release) Macro {
 		GroupID:             release.GroupID,
 		InfoUrl:             release.InfoURL,
 		DownloadUrl:         release.DownloadURL,
-		Indexer:             release.Indexer,
+		Indexer:             release.Indexer.Identifier,
```

Comment:
Should this one be just `release.Indexer.Id`?

Reply:
Ideally it could be the full struct but that would break filters that already use and expect the identifier. We could do a db migration tho 🤔